src/components/radio-button/radio-button-test.stories.tsx

```diff
@@ -131,3 +138,58 @@ export const WithTooltipPositionOnRadioGroup: ComponentStory<
 
 WithTooltipPositionOnRadioGroup.storyName =
   "with tooltip position on RadioGroup";
+
+const radioContainerWidth = 400;
+
+export const RadiobuttonComponent = ({ ...props }) => {
+  const [isChecked, setIsChecked] = React.useState(false);
+  return (
+    <>
+      <div
+        style={{
+          marginTop: "64px",
+          marginLeft: "64px",
+          width: radioContainerWidth,
+        }}
+      >
+        <RadioButton
+          id="radio-1"
+          value="radio1"
+          label="Radiobutton 1"
+          // checked={() => setIsChecked(false)}
+          checked={isChecked}
+          onChange={(e) => setIsChecked(e.target.checked)}
+          {...props}
+        />
+      </div>
+    </>
+  );
+};
+
+export const RadiobuttonGroupComponent = ({
```

Comment:
I think this should be `RadioButtonGroupComponent` in line with the rest of the naming convention.